onchain/wallet/wallet_test.go

```diff
@@ -82,6 +82,13 @@ func TestImportSlip132Wallet(t *testing.T) {
 				CoreDescriptor: "wpkh([72411c95/84'/1'/0']tpubDC2Q4xK4XH72JQHXbEJa4shGP8ScAPNVNuAWszA2wo6Qjzf4zo2ke69SshBpmJv8CKDX76QN64QPiiSJjC69hGgUtV2AgiVSzSQ6zgpZFGU/1/*)#tv66wgk5",
 			},
 		},
+		{
+			"CoreDescriptor",
+			onchainWallet.Credentials{
+				CoreDescriptor: "ct(slip77(099d2fa0d9e56478d00ba3044a55aa9878a2f0e1c0fd1c57962573994771f87a),elwpkh([a2e8a626/84'/1'/0']tpubDC2Q4xK4XH72HUSL1DTS5ZCyqTKGV71RSCYS46eE9ei45qPLFWEVNr1gmkSXw6NCXmnLdnCx6YPv5fFMenHBmM4UXfPXP56MwikvmPFsh2b/0/*))#60v4fm2h",
+				Currency:       boltz.CurrencyLiquid,
```

Comment:
This is broken

Reply:
the artifacts against which the ci runs are still the old gdk version